assembl/scripts/ini_files.py

```diff
@@ -177,6 +177,7 @@ def generate_ini_files(config, config_fname):
             'autostart_webpack',
             'autostart_elasticsearch_server',
             'autostart_urlmetadata',
+            'autostart_bluenove_actionable',
```

Comment:
I saw recently there is a list around line 62 with default values, can you please add autostart_urlmetadata and autostart_bluenove_actionable there? Thanks.

Reply:
old code ;)